packages/sakuli-legacy/src/loader/legacy-project-properties.class.ts

```diff
@@ -79,6 +79,44 @@ export class LegacyProjectProperties {
     return this.browser || this.testsuiteBrowser || "firefox";
   }
 
+  /**
+   * Defines whether the browser should be reused after a test suite or a new browser session should be created
+   *
+   * To access the actual set browser within Sakuli use [isReuseBrowser()]{@link LegacyProjectProperties.isReuseBrowser}
+   */
+  @BooleanProperty("reuseBrowser")
+  reuseBrowser: Maybe<boolean>;
+
+  /**
+   * Defines whether the browser should be reused after a test suite or a new browser session should be created
+   *
+   * To access the actual set browser within Sakuli use [isReuseBrowser()]{@link LegacyProjectProperties.isReuseBrowser}
+   */
+  @BooleanProperty("testsuite.reuseBrowser")
+  testsuiteReuseBrowser: Maybe<boolean>;
```

Comment:
Why is this property required? If I understand the test case `should return browserReuse if set even testsuiteBrowserReuse is set`, it is overwritten anyways and just a fallback. I'd also reduce the added config options to an absolute minimum. It was required in the acceptance criteria to recreate browser sessions after a test case by global configuration. As long as no one asks for test suite based config, this is over engineering.